### PR TITLE
Create a series of NFT tutorials

### PR DESCRIPTION
**Pull request by bucanero**
A draft of a bare-bones NFT tutorial

Fixes https://github.com/near/devrel/issues/348

### CODE REVIEW

File: docs/tutorials/contracts/nfts/3-enumeration.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+This should return an output similar to the following:
+
+<details>
+<summary>Example response: </summary>
+<p>
+
+```json
+[
+]
```
Comment (BenKurrek) [inline docs/tutorials/contracts/nfts/3-enumeration.md:110]:
```suggestion
0
```

File: docs/tutorials/contracts/nfts/3-enumeration.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+</p>
+</details>
+
+## Conclusion
+
+In this tutorial, you have added two [new enumeration functions](/docs/tutorials/contracts/nfts/enumeration#modifications-to-the-contract), and now you have a basic NFT smart contract with minting and enumeration methods in place. After implementing these modifications, you redeployed the smart contract and tested the functions using the CLI.
+
+In the next tutorial, you'll implement the core functions needed to allow users to transfer the minted tokens.
+
+:::info Remeber
```
Comment (BenKurrek) [inline docs/tutorials/contracts/nfts/3-enumeration.md:122]:
```suggestion
:::info Remember
```

**Comment by danielwpz** [comment]
this is a great tutorial!

**Comment by BenKurrek** [comment]
> this is a great tutorial!

Thanks @danielwpz! :)

**Pull request merged**